locale/el_GR/admin.xml

```diff
@@ -1,77 +1,131 @@
-	<message key="admin.serverInformation">Πληροφορίες Server </message>
-	<message key="admin.server.phpVersion">Έκδοση PHP </message>
-	<message key="admin.server.platform">Πλατφόρμα λειτουργικού συστήματος</message>
-	<message key="admin.settings.about">Σχετικά με τον ιστότοπο</message>
-	<message key="admin.settings.contactEmail">Email Υπεύθυνου Επικοινωνίας</message>
-	<message key="admin.settings.contactName">Όνοματεπώνυμο Υπεύθυνου Επικοινωνίας</message>
-	<message key="admin.settings.introduction">Εισαγωγή</message>
-	<message key="admin.settings.minPasswordLength">Ελάχιστο μήκος κωδικού πρόσβασης (password)</message>
-	<message key="admin.settings.passwordCharacters">χαρακτήρες</message>
-	<message key="admin.settings.siteLanguage">Γλώσσα ιστοτόπου</message>
-	<message key="admin.settings.siteStyleSheetInvalid">Μη έγκυρη μορφή αρχείου stylesheet ιστοτόπου. Έγκυρη μορφή αρχείου είναι η  .css.</message>
-	<message key="admin.settings.siteStyleSheet">Style sheet ιστοτόπου</message>
-	<message key="admin.settings.siteTitle">Τίτλος Ιστότοπου</message>
-	<message key="admin.siteAdmin">Διαχειριστής Ιστοτόπου (Συλλογής)</message>
-	<message key="admin.siteManagement">Διαχείριση Ιστοτόπου</message>
-	<message key="admin.siteSettings">Ρυθμίσεις Ιστοτόπου</message>
-	<message key="admin.systemConfigFileReadError"><![CDATA[Το αρχείο  ρυθμίσεων <tt>config.inc.php</tt> δεν υπάρχει, δεν είναι  αναγνώσιμο, ή δεν είναι έγκυρο.]]></message>
-	<message key="admin.systemInformation">Πληροφορίες Συστήματος</message>
-	<message key="admin.versionBuild">Έκδοση</message>
-	<message key="admin.version.checkForUpdates">Ελέγξτε για  ενημερώσεις</message>
-	<message key="admin.version.downloadPackage">Κατέβασμα</message>
-	<message key="admin.version.downloadPatch">Κατέβασμα προγράμματος επιδιόρθωσης (patch)</message>
-	<message key="admin.versionHistory">Ιστορικό Έκδοσης</message>
-	<message key="admin.version.latest">Τελευταία έκδοση</message>
-	<message key="admin.versionMajor">Κύρια</message>
-	<message key="admin.versionMinor">Δευτερεύουσα</message>
-	<message key="admin.version.moreInfo">Περισσότερες  πληροφορίες</message>
-	<message key="admin.versionRevision">Αναθεώρηση</message>
-	<message key="admin.version.updateAvailable">Μια νεώτερη  ενημερωμένη έκδοση είναι διαθέσιμη</message>
-	<message key="admin.version.upToDate">Το σύστημά σας είναι  ενημερωμένο με την τελευταία έκδοση</message>
-	<message key="admin.version">Έκδοση</message>
-</locale>
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<!DOCTYPE locale SYSTEM "../../dtd/locale.dtd">
+
+<!--
+  * locale/el_GR/admin.xml
+  *
+  * Copyright (c) 2014-2018 Simon Fraser University
+  * Copyright (c) 2000-2018 John Willinsky
+  * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+  *
+  * Localization strings.
+  *
+  * Edited by National Documentation Center (EKT) team, 2019
+  *
+  -->
+
+  <locale full_name="Greek" name="el_GR">

```

Comment:
The full name should be "ελληνικά", and I would be happy if you could change the order back to `<locale name="el_GR"> full_name="ελληνικά">`